### DISCUSSION TITLE
KEYCDN_USER_AGENT 参数设置

### DISCUSSION THREAD

**Discussion by NoverNobida** [jason5ng32/MyIP#153] [Q&A] [answered]
KEYCDN_USER_AGENT 参数应当如何设置呢？我使用自己浏览器得到的结果 `KEYCDN_USER_AGENT='Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0'` 似乎并没有用。

**Comment by jason5ng32** [comment, marked as answer]
这个变量我的命名有些误导性，实际上需要填写的是你请求时使用的域名。已经在 Readme 中修正。

**Reply by NoverNobida** [reply]
更改配置后运行良好。
另外，是否有计划进一步提高自建的灵活性？比如自定义获取ip信息的api，开源ptest-1.ipcheck.ing 测试网址等。

**Reply by jason5ng32** [reply]
No.
